python/spark_sklearn/grid_search.py

```diff
@@ -124,12 +121,12 @@ class GridSearchCV(BaseSearchCV):
     ...                             # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
     GridSearchCV(cv=None, error_score=...,
            estimator=SVC(C=1.0, cache_size=..., class_weight=..., coef0=...,
-                         decision_function_shape=None, degree=..., gamma=...,
+                         decision_function_shape=..., degree=..., gamma=...,
                          kernel='rbf', max_iter=-1, probability=False,
                          random_state=None, shrinking=True, tol=...,
                          verbose=False),
-           fit_params={}, iid=..., n_jobs=1,
-           param_grid=..., pre_dispatch=..., refit=..., return_train_score=...,
+           fit_params=..., iid=..., n_jobs=1,
```

Comment:
This is fine. Just wondering since the [original code in scikit-learn](https://github.com/scikit-learn/scikit-learn/blob/0.18.1/sklearn/model_selection/_search.py#L791-L800) uses `fit_params={}` in the docstring, is our doctest somewhat different from theirs?

Reply:
Our doctest (as of your review) didn't actually test the spark-sklearn GridSearchCV implementation; it instead tested against scikit-learn's GridSearchCV implementation. I've since updated it to run against the spark-sklearn implementation.